app/assets/src/components/views/bulk_download/ChooseStep.jsx

```diff
@@ -479,25 +416,73 @@ class ChooseStep extends React.Component {
     );
   };
 
-  renderFilteredSamplesWarning = () => {
-    const { filteredSampleNames } = this.props;
+  renderInvalidSamplesWarning = () => {
+    const { invalidSampleNames } = this.props;
+
+    const header = (
+      <div className={cs.header}>
+        <span className={cs.highlight}>
+          {invalidSampleNames.length} sample
+          {invalidSampleNames.length > 1 ? "s" : ""} won't be included in the
+          bulk download
+        </span>, because they are in progress or failed samples:
+      </div>
+    );
+
+    const content = (
+      <div className={cs.messageContainer}>
+        {invalidSampleNames.map((name, index) => {
```

Comment:
I would recommend passing the list of names (as an array of strings) to CompactListNotification, instead of rendering the list here and then passing the `messageContainer` node to CompactListNotification.

If the purpose of the compact "list" notification is to display a list of items, then I would pass the list and let the component figure out how to render the list (instead of rendering it here).

You can also put the styling for `messageContainer` and `messageLine` in CompactListNotification.

Reply:
What if I just changed the name the `CompactNotification` or something similar? I thought about doing this, but I decided that having more flexibility over what to display in the toggleable container would be more useful.

Reply:
That would also work! I would call it `ExpandableNotification` or `AccordionNotification` (since it uses an Accordion). I think you should still move some of the styling into this component, rather than have the caller do the styling. (styling such as the left-margin and the background color, as we discussed)